src/nav/MainNavBar.js

```diff
@@ -61,14 +55,7 @@ class MainNavBar extends PureComponent {
           onPress={leftPress}
         />
         <Title color={textColor} />
-        {!editMessage &&
-          <NavButton
-            name="md-people"
-            color={textColor}
-            borderRadius={20}
-            unreadCount={unreadPmsTotal + unreadHuddlesTotal}
-            onPress={onPressPeople}
-          />}
+        <NavButton placeholder />
```

Comment:
@borisyankov having a placeholder component did with making a number of changes to the styles of core components. Is this a plausible solution?